README.md

```diff
@@ -53,12 +54,13 @@ As you begin contributing to an increasing amount of GitHub repositories, you'll
 
 ## 📚 Usage
 
-### ⚙️ `fgh configure`
+### 🔒 `fgh login`
+
+Before using `fgh`, you'll need to give it access to your GitHub account. Simply run `fgh login` to quickly get set up!
```

Comment:
Could we add a note saying that users could use a PAT if they can't get OAuth access (organizations permissions for example)? Is it a good idea to point out where the token is stored on a user's machine?